### PR TITLE
Create 732 - Stop going to Spam.md

### PR DESCRIPTION
**Pull request by randyrektor**

### CODE REVIEW

File: shows/732 - Stop going to Spam.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+number: 732
+title: "Stop going to Spam: DMARC, SPF and DKIM Explained"
+date: 1708344000810
+url: https://traffic.libsyn.com/syntax/Syntax_-_732.mp3
+youtube_url: https://youtu.be/ERiQc8hoDvo
```
Comment (wesbos) [inline shows/732 - Stop going to Spam.md:6]: @randyrektor Can we starrt using youtube.com urls directly? Its a small thing, but probably helps the SEO google juice linking directly to the page without a redirect

File: shows/732 - Stop going to Spam.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+Wes and Scott explore the web standards and email authentication methods that keep your email from going to spam; DMARC, SPF and DKIM. If you send email via a custom domain name, you need to listen to this!
+
+### Show Notes
+
+* **[00:00](#t=00:00)** Welcome to Syntax!
+* **[01:06](#t=01:06)** Brought to you by Sentry.io.
```
Comment (wesbos) [inline shows/732 - Stop going to Spam.md:14]: We should link Sentry up in the show notes as well - their SEO team likes this

Comment (randyrektor) [reply shows/732 - Stop going to Spam.md:14]: Ahhh great catch. Will do

**Comment by stolinski** [comment]
I re-ran the test and it passed. Wonder why that one is flaky.

**Comment by wesbos** [comment]
might need to await the mp3 load if we aren't doing that already.

**Comment by stolinski** [comment]
Yeah there might be something there, but playwright should be waiting already. The worst kind of annoying test is the one that only sometimes fails.

**Pull request merged**